packages/jest-cli/src/__tests__/__snapshots__/watch.test.js.snap

```diff
@@ -5,6 +5,7 @@ Array [
   "
 Watch Usage
  › Press a to run all tests.
+ › Press f to enable \\"only-failures\\" mode.
```

Comment:
Should we phrase it in a way that is more similar to "to run only failed tests" so that we don't expose the "only-failures" keyword?